pipeline/beam_tables.py

```diff
@@ -123,7 +123,7 @@ def _add_schemas(schema_a: Dict[str, Any],
     BASE_BIGQUERY_SCHEMA, {
```

Comment:
We may want to discuss the schema in a doc instead, since we can't comment on the base schema and explore suggestions here easily.

Reply:
Yes. I didn't make this intending to discuss the schema here, it was just a placeholder for the future.